omigami/spectra_matching.py

```diff
@@ -133,6 +135,16 @@ def _build_payload(
         """
         spectra = []
         for spectrum in batch:
+            if "pepmass" in spectrum.metadata:
+                precursor_mz = str(spectrum.metadata["pepmass"][0])
+            elif "precursor_mz" in spectrum.metadata:
+                precursor_mz = str(spectrum.metadata["precursor_mz"][0])
+            else:
+                raise KeyError(
+                    "One of ['pepmass' or 'precursor_mz'] must be present in the"
+                    "spectrum metadata."
+                )
```

Comment:
this is for this task, right? https://datarevenue.atlassian.net/browse/OMI-458

Reply:
yes. It was a small change I included it here